package.json

```diff
@@ -11,7 +11,7 @@
     }
   },
   "scripts": {
-    "postinstall": "patch-package && if [ \"$NODE_ENV\" != \"production\" ]; then npx crlf --set=LF ./node_modules/graphql-schema-typescript/**/*; fi",
+    "postinstall": "patch-package",
```

Comment:
I moved the LF fix to `gateway` package as the dependency moved under its dependencies